core/primitives/src/transaction.rs

```diff
@@ -61,8 +140,6 @@ pub struct TransactionV1 {
     pub block_hash: CryptoHash,
     /// A list of actions to be applied
     pub actions: Vec<Action>,
-    /// Priority fee. Unit is 10^12 yoctoNEAR
-    pub priority_fee: u64,
```

Comment:
priority is also a part of the receipt, please add a `TODO(gas-keys)` to remove it later